### DISCUSSION TITLE
Chasse aux objets collectionnables

### DISCUSSION THREAD

**Discussion by Syandre** [Crownicles/Crownicles#2130] [Suggestions fermées]
Une idée d'objectif secondaire plus ou moins passif, à réaliser sur le long terme.

Le but serait d'ajouter un genre de carnet ou sac de collection dans lequel il serait possible d'ajouter différents objets trouvés pendant les voyages, évents, voir chez certains marchands, de manière aléatoire ou liés à certains lieux avec des taux d'obtention variés afin de corser un peu les chances d'obtention.

Cela pourrait se présenter sous différentes catégories de familles d'objets liés aux émojis de Discord à rassembler, avec une petite récompense une fois complétée, en argent, points, expérience voir pourquoi pas en très légers bonus de statistiques.

**Exemples rapides :**
> - **Collection : Figurines**
> 👮 Figurine d'agent du royaume/villes 🧝‍♀️ Figurine d'elfe 🕵️ Figurine de détective 👨‍🌾 Figurine de fermier 👨‍⚕️... 👷... 👨‍🍳...  [....] 
> _Gain d'un peu d'argent, de points, d'expérience une fois complétée._
> 
> - **Collection : Armes**
> 🏹 Arc de collection 🗡️ Dague de collection ⚔️ Épée de collection 🍌 Sabre banane de collection 💣... 🪓...  [....]
> _Connaissances en armes, gain de +1🗡️ en valeur d'attaque une fois complétée._
> 
> - **Collection : Protections**
> 🛡️ Bouclier de collection 🦾 Coudières de collection 🧥 Cape d'un voleur aujourd'hui en prison 👚... 🥼... ⛑️...  [....]
> _Connaissance des équipements défensifs. Gain de +1🛡️ de valeur de défense une fois complétée._
> 
> - **Collection : Légumes**
> 🥕 Carotte de collection 🥦 Brocoli géant 🥑 Avocat impressionnant 🍆... 🥬... 🥒... [...longue liste encore]
> _Connaissances en alimentation +1❤️ à votre maximum de santé une fois complétée._
>
>- **Collection : Badges**
>  🔴Badge rouge 🟢Badge vert 🟪 Badge carré violet 🔵 Badge de Stitch...? 🟨... 💛... 🟦... 🟨... [...]
> _Gain de quelques points🏅 une fois complétée._
>
> - **Des tas d'autres collections :**
> Plein, plein, plein, que ce soit long à rassembler !

Les options de familles d'objets ne manquent pas, les émojis non plus, au même titre que pour les équipements actuels.

Serait-ce cependant quelque chose gérable au vu de la QUANTITÉ d'objets que ça impliquerait de créer/ajouter/gérer ? 
Ou ne serait-ce que pour l'affichage de "l'inventaire de collection actuel" sur Discord sans que ce ne soit qu'un gros bazar illisible ?
A vous de dire. J'imagine bien que ce ne serait pas qu'un ajout mineur à réaliser non plus en terme de travail.

**Comment by BananePlantain** [comment]
J'aime bien l'idée mais qu'entends tu par "connaissances en ..."?

**Reply by Syandre** [reply]
Rien qu'un texte d'exemple de blabla sur le pourquoi on gagnerait des points, de l'expérience ou +1 sur une statistique une fois une collection complétée. Généralement lorsqu'on collectionne quelque chose et commence à accumuler, on fini par s'y connaitre un petit peu sur le sujet. Si on collectionne les armes, on sait un peu comment s'en servir ou bien leur fonctionnement (+1🗡️). Si on collectionne les pierres, on fini par savoir pourquoi elles ont telle couleur selon leur composition, et les gens font "wow belle collection" (🎖️ gain de réputation).

En vrai ça pourrait être autre chose de moins pompant, voir rien du tout si trop encombrant en texte.

**Reply by BananePlantain** [reply]
ou alors  connaissances en arme=moins de chance de rater une attaque,connaissances en bouclier plus de succès de l'action "se défendre" lors du mini FightPet... mais a chaque fois juste un avantage léger

**Reply by BastLast** [reply]
Yo juste pour dire en ayant une vue sur le backlog je pense que cette idée est trop complexe à développer par rapport à ce qu'elle apporte au niveau du gameplay et tu l'envoies préférés concentré de nos efforts sur des choses plus simples à développer ou sur les choses déjà complexes que nous avons déjà prévu du coup je déplace cette idée d'en refuser pour cette raison
